activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Add `unfreeze_time` alias method to `travel_back` in `ActiveSupport::Testing::TimeHelpers`.
```

Comment:
I think "alias method to" is not correct grammatically (though I am not a native speaker). Could be something like

> Define `unfreeze_time` as an alias of `travel_back` in `ActiveSupport::Testing::TimeHelpers`.

for example.

Reply:
That grammar change is now updated in the CHANGELOG.